victimsdb_lib/model.py

```diff
@@ -0,0 +1,170 @@
+"""VictimsDB model."""
+
+import re
+import yaml
+from f8a_version_comparator.comparable_version import ComparableVersion
+
+from victimsdb_lib.errors import ParseError
+
+
+class Record(object):
+    """CVE Record in a database."""
+
+    def __init__(self, cve_id, title, description, cvss_v2, references, affected):
+        if cve_id:
+            cve_id = cve_id.strip()
+            if not cve_id.startswith('CVE-'):
+                cve_id = 'CVE-{cve_id}'.format(cve_id=cve_id)
+        self.cve_id = cve_id
+        self.title = title or ''
+        self.description = description or ''
+        self.cvss_v2 = cvss_v2 or None
```

Comment:
tbh I don't understand why `or None`, but well why not, you are explicit here, so ok :)